src/View/view.js

```diff
@@ -0,0 +1,31 @@
+import { Edit, Remove } from '../Actions/actions.js';
+import ElementCreator from '../Controller/ElementCreator.js';
+
+function Appendage(el, child, others) {
+  const [styles, tasks, kebab] = others;
+  const { description, id } = tasks;
+  const DomEntry = document.getElementById('entry');
+  const childElement = ElementCreator(child);
+
+  const box = ElementCreator('input');
+  const edit = ElementCreator('input');
+  const img = ElementCreator('img');
+  img.setAttribute('src', kebab);
+  box.type = 'checkbox';
```

Comment:
- [ ] By default, new tasks should have the property completed set to `false` but when you check the box it should be `true` but currently when I check the box it doesn't change please address that

Reply:
Thanks, @DeliceLydia for your kind feedback. I must say that the required changes you requested are supposed to be effected in my next exercise. This is happening because I failed to submit the previous day's exercise and so submitting it today. I will effect them all the same but will have to submit the same code base for review in my next exercise too. Please let me know what you advise on this issue. Regards.